app/src/main/java/de/langerhans/odintools/tools/SettingsRepo.kt

```diff
@@ -0,0 +1,110 @@
+package de.langerhans.odintools.tools
+
+import de.langerhans.odintools.BuildConfig
+import javax.inject.Inject
+
+class SettingsRepo @Inject constructor(
+    private val executor: ShellExecutor,
+) {
+
+    init {
```

Comment:
I'd make this call explicitly somewhere where it makes most sense, e.g. OdinToolsApplication. A place that's only run once at startup. Right now this would be called for every class that injects an instance of SettingsRepo right?

Reply:
Actually, this is what the `println('Settings init') was meant for. Wanted to confirm if SettingsRepo was being initialized for every class that injects an instance. Not sure if this is the best way to confirm this, but can agree that making this call explicitly would make more sense.